depthai_nodes/ml/parsers/utils/yolo.py

```diff
@@ -173,8 +182,10 @@ def non_max_suppression(
 def parse_yolo_outputs(
     outputs: List[np.ndarray],
     strides: List[int],
-    anchors: np.ndarray,
+    anchors: List[Optional[np.ndarray]],
     kpts: Optional[List[np.ndarray]] = None,
+    det_mode: bool = False,
+    yolo_version: Optional[str] = None,
```

Comment:
Is YoloP parsing any different than other Yolo version with anchors (e.g. YoloV5, YoloV4, YoloV3)? Do we need to specify `yolo_version` or is the difference only if it's anchor or anchor-less model (from YoloV6 and newer)?

Basically can we only check if anchors are supplied and then if they are use them, otherwise go with the other postprocessing?